backend/src/federation/index.ts

```diff
@@ -36,19 +32,61 @@ export const startDHT = async (
     logger.info(`keyPairDHT: publicKey=${keyPair.publicKey.toString('hex')}`)
     logger.debug(`keyPairDHT: secretKey=${keyPair.secretKey.toString('hex')}`)
 
+    const apiList = {
+      apiVersions: Object.values(ApiVersionType).map(function (apiEnum) {
+        return { api: apiEnum, url: CONFIG.FEDERATION_COMMUNITY_URL }
```

Comment:
Deliver the "correct" URL which should include the version number.
```suggestion
        return { api: apiEnum, url: CONFIG.FEDERATION_COMMUNITY_URL + apiEnum + '/' }
```

Reply:
the concatination for a complete usable url should not be done at receiving time, instead it will be done in the client class at the point of time where it is realy used for sending a request.